assets/js/broken_redirect_list.js

```diff
@@ -879,6 +879,7 @@ validurls['/docs/user_guide/sage_ai/intelligence/faqs'] = '/docs/user_guide/braz
 validurls['/docs/user_guide/sage_ai/recommendations/'] = '/docs/user_guide/brazeai/recommendations/';
 validurls['/docs/user_guide/sage_ai/recommendations/about_item_recommendations/'] = '/docs/user_guide/brazeai/recommendations/about_item_recommendations/';
 validurls['/docs/user_guide/sage_ai/recommendations/ai_item_recommendations/'] = '/docs/user_guide/brazeai/recommendations/ai_item_recommendations/';
+validurls['/docs/user_guide/brazeai/recommendations/#property-name'] = '/docs/user_guide/brazeai/recommendations/ai_item_recommendations/#property-name';
```

Comment:
```suggestion
validurls['/docs/user_guide/brazeai/recommendations/#property-name'] = '/docs/user_guide/brazeai/recommendations/ai_item_recommendations#property-name';
```

i believe its due to the extra slash before the `#` char.